examples/simple-nft-minter/index.mjs

```diff
@@ -0,0 +1,72 @@
+import { Reach, test } from '@reach-sh/stdlib';
+const stdlib = new Reach(process.env, { REACH_NO_WARN: 'Y' });
+
+const startingBalance = stdlib.parseCurrency(100);
+const [minter, receiver] = await stdlib.newTestAccounts(2, startingBalance);
+
+const gasLimit = 500000;
+if (stdlib.connector != 'ALGO') {minter.setGasLimit(gasLimit) && receiver.setGasLimit(gasLimit)};
+
+const mintAddr = minter.getAddress();
+console.log(`Minter's address is ${mintAddr}`);
+const recAddr = receiver.getAddress();
+console.log(`Receiver's address is ${recAddr}`);
+
+const minterAddrFormat = await stdlib.formatAddress(minter);
+console.log(`The minter's formatted address is ${minterAddrFormat}`);
+const receiverAddrFormat = await stdlib.formatAddress(receiver);
+console.log(`The receiver's formatted address is ${receiverAddrFormat}`);
+
+const fmt = (x) => stdlib.formatCurrency(x, 4);
+const getBal = async (who, tok) => tok ? (await stdlib.balanceOf(who, tok)) : fmt(await stdlib.balanceOf(who));
+
+const logBalance = async (acc, tok, accStr = acc.getAddress()) => {
+    const bal = await getBal(acc, tok, accStr);
+    const unit = tok ? 'of the NFT' : stdlib.standardUnit;
+    console.log(`${accStr} has ${bal} ${unit}.`);
```

Comment:
I just made an addition to the stdlib that should be released tomorrow: `acc.getDebugLabel`.

So, when you create the accounts you can set the debug label
```JavaScript
minter.setDebugLabel('minter');
```

and this function can be cleaned up to:
```JavaScript
const logBalance = async (acc, tok) => {
    const bal = await getBal(acc, tok, accStr);
    const unit = tok ? 'of the NFT' : stdlib.standardUnit;
    console.log(`${acc.getDebugLabel()} has ${bal} ${unit}.`);
    return bal;
}
```